uber-core/src/main/java/com/uber/sdk/core/auth/AccessTokenAuthenticator.java

```diff
@@ -61,7 +61,10 @@ public AccessTokenAuthenticator(SessionConfiguration sessionConfiguration,
 
     @Override
     public void signRequest(Request.Builder builder) {
-        setBearerToken(builder, tokenStorage.getAccessToken());
+        if(tokenStorage.getAccessToken() != null && tokenStorage.getAccessToken().getToken() !=
```

Comment:
Might be a good idea to add nullability annotations for `AccessToken`; noticed it in other areas of code. I don't think the model makes sense to have an access token object that has a null `getToken` getter.

Reply:
Will address after future nullability convo.